### PR TITLE
Add changelog

### PR DESCRIPTION
**Pull request by emolsson**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Changes
+
+## Version 2.0
```
Comment (eperott) [inline CHANGES.md]: Are we already moving towards a new major version? I.e, are there any non-bwc changes in the list. I suppose the filename changes qualify...?

With semantic versioning, shouldn't there be 3 positions in the version number?

File: CHANGES.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+* Dynamic license header check - #37
```
Comment (eperott) [inline CHANGES.md]: Does it make sense to include "everything" here such as these things, or should we keep it down to "user facing" changes?

File: CHANGES.md (hunk 1 above)
Comment (eperott) [reply CHANGES.md]: pom files indicate we're moving towards version 1.0.1

Comment (emolsson) [reply CHANGES.md]: Changed pom versions to 2.0.0 & updated the changlog with 2.0.0

File: CHANGES.md (hunk 2 above)
Comment (emolsson) [reply CHANGES.md]: I think user/dev facing changes should be mentioned. I.e. not updating karaf test version, but if we change an interface.

Comment (eperott) [reply CHANGES.md]: Sounds reasonable.

Another way to slice would be to keep user facing stuff only in the changelog, and let developers use git log. No strong opinion though.

**Pull request merged**